app/components/request_form/request_form.js

```diff
@@ -15,11 +15,22 @@ export default class extends Controller {
     this.updateMembersCount();
   }
 
+  insertPlaceholderAtCursor() {
+    const placeholder = '{{VORNAME}}';
+    const [start, end] = [
+      this.messageTarget.selectionStart,
+      this.messageTarget.selectionEnd,
+    ];
+    this.messageTarget.setRangeText(placeholder, start, end, 'end');
+    const event = new Event('input', { bubbles: true });
+    this.messageTarget.dispatchEvent(event);
+    this.messageTarget.focus();
+  }
+
   updatePreview() {
     let message = sanitize(this.messageTarget.value);
     message = message || this.previewFallbackValue;
-    message = replacePlaceholder(message, 'FIRST_NAME', 'Max');
-
+    message = replacePlaceholder(message, 'VORNAME', 'Max');
```

Comment:
maybe refactor 'VORNAME' into a placeholder variable like in `insertPlaceholderAtCursor`

Reply:
sure, good idea!